src/tableutils.js

```diff
@@ -789,6 +765,69 @@ function updateHeadingRows( table, first, last, model, batch ) {
 	}
 }
 
+// Finds cells that will be:
+// - trimmed - Cells that are "above" removed rows sections and overlaps removed section - their rowspan must be trimmed.
+// - moved - Cells from removed rows section might stick out of. Such cells are moved to a next row after a removed section.
+//
+// Sample table with overlapping & sticking out cells:
+//
+//      +----+----+----+----+----+
+//      | 00 | 01 | 02 | 03 | 04 |
+//      +----+    +    +    +    +
+//      | 10 |    |    |    |    |
+//      +----+----+    +    +    +
+//      | 20 | 21 |    |    |    | <-- removed row
+//      +    +    +----+    +    +
+//      |    |    | 32 |    |    | <-- removed row
+//      +----+    +    +----+    +
+//      | 40 |    |    | 43 |    |
+//      +----+----+----+----+----+
+//
+// In a table above:
+// - cells to trim: '02', '03' & '04'.
+// - cells to move: '21' & '32'.
+function getCellsToMoveAndTrimOnRemoveRow( table, last, first, rowsToRemove ) {
```

Comment:
Order of arguments is confusing, why not like this:
```
function getCellsToMoveAndTrimOnRemoveRow( table, first, last )
```
Also `rowsToRemove` seems redundant.

Reply:
Yeah - I don't wanted to recalculate this but it is in one place only and with a comment so I removed that.

The order of arguments was also fixed.